src/nvim/tui/tui.c

```diff
@@ -954,6 +964,15 @@ static TermType detect_term(const char *term, const char *colorterm)
   if (colorterm && strstr(colorterm, "gnome-terminal")) {
     return kTermGnome;
   }
+  if (STARTS_WITH(term, "xterm")) {
+    return kTermXTerm;
+  }
+  if (STARTS_WITH(term, "dtterm")) {
+    return kTermDTTerm;
+  }
+  if (STARTS_WITH(term, "teraterm")) {
+    return kTermTeraTerm;
+  }
```

Comment:
Unless you have plans to use this granularity in a future patch, let's consider all of these to be xterm.